public/js/p3/widget/SelectionToGroup.js

```diff
@@ -108,36 +109,82 @@ define([
 		onCopy: function(evt){
 			// console.log("Copy Selection: ", this.selection, " to ", this.value);
 			//var idType = (this.type == "genome_group") ? "genome_id" : "feature_id"
-			if(!this.idType){
-				this.idType = "genome_id";
-				if(this.type == "genome_group"){
+
+			// for subsystems grab all associated features in async call
+			if (this.selection[0] && this.selection[0].hasOwnProperty("subsystem_id")) {
```

Comment:
okay, lets discuss this part. I think we should move the querying part to Subsystem grid container. That is how other grids are working (such as protein family sorter).